src/main/ionic/peers/src/pages/my-profile/my-profile.html

```diff
@@ -0,0 +1,47 @@
+<ion-header>
+  <ion-navbar>
+    <button ion-button menuToggle>
+      <ion-icon name="menu"></ion-icon>
+    </button>
+    <ion-title>My Profile</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
+  <ion-list>
+    <button ion-item *ngFor="let item of items" (click)="itemTapped($event, item)">
+      <ion-icon [name]="item.icon" item-start></ion-icon>
+      {{item.title}}
+      <div class="item-note" item-end>
+        {{item.note}}
+      </div>
+    </button>
+  </ion-list>
+  <div *ngIf="selectedItem" padding>
+    You navigated here from <b>{{selectedItem.title}}</b>
+  </div>
+  <ion-grid *ngIf="user">
+    <ion-row>
+      <ion-col col-md-4 offset-md-4 col-xs-6 offset-xs-3>
```

Comment:
I think `col-md-6 offset-md-3` is better. The col-xs doesn't seem to have any effect anyway as it's still full width on my machine. But I think we want it full width on small screens, so you can remove the xs parts.